content/redirect/pipeline-snippet-generator.adoc

```diff
@@ -0,0 +1,4 @@
+---
+layout: redirect
+redirect_url: https://www.jenkins.io/doc/book/pipeline/getting-started/#snippet-generator
```

Comment:
I believe the preference in other files is to use a link that is within the site rather than including the hostname of the site in the redirect URL.
```suggestion
redirect_url: /doc/book/pipeline/getting-started/#snippet-generator
```

Reply:
Yeah I wasn't sure about whether to use an absolute URL or not here. Should that guidance be added to https://github.com/jenkins-infra/jenkins.io/blob/master/CONTRIBUTING.adoc#adding-a-redirect-page?